### DISCUSSION TITLE
What is backlinks?

### DISCUSSION THREAD

**Discussion by Mayb3Nots** [isar/isar#326] [Help] [answered]
Ask the title states I don't quite get what is backlinks even reading from the docs and googling it doesn't provide a full answer. From the example `We want to know which students a specific teacher has so we define a backlink:` does that mean backlink-ing is for querying? If so how does it work?

**Comment by simc** [comment, marked as answer]
Backlinks are reversed links. So if you have object `A` and `B` and you have a link A->B:
A1 -> B1
A2 -> [B1, B2]

The backlink will be B->A:
B1 -> [A1, A2]
B2 -> A2

And yes, you can use them for querying

**Comment by hokkar** [comment]
https://moessmartb2b.com/ this is!
